src/Frost.ts

```diff
@@ -331,14 +279,25 @@ export class Frost {
 
     const request = fetch(`${this.host}${Path.ACCOUNTS_PROFILE}`, options)
 
-    return Promise.race([request, this.timeoutPromise()])
-      .then(async (value: any) => {
-        if (value.ok) return await value.json()
+    return Promise.race([request, this.timeoutPromise()]).then(async (value: any) => {
+      if (value.ok) return await value.json()
+      throw await value.text()
+    })
+  }
 
-        throw await value.text()
-      })
-      .catch(e => {
-        throw e
-      })
+  getPublicKeys(cuid?: string): Promise<[string]> {
```

Comment:
`[string]` is [a tuple type](https://www.typescriptlang.org/docs/handbook/basic-types.html). 

It's declaring that this method always returns an array with a single element. 

Is this the intended behavior?

If we want to specify that it returns an array, the type annotation should be `Promise<ReadonlyArray<string>>` instead.

Reply:
An array of strings, of any length. `string[]` is shorthand for `Array<string>`, but `ReadonlyArray` doesn't have a shorthand version.

Thinking again about it, though, it shouldn't return a ReadonlyArray, so `Promise<string[]>` is ok.